### DISCUSSION TITLE
installation

### DISCUSSION THREAD

**Discussion by ghost** [Rolf-Hempel/PlanetarySystemStacker#69] [General]
Any chance of a Windows installer .exe like in version 8.3 ?

**Comment by Rolf-Hempel** [comment]
The problem is related to the software "pyinstaller" which I used to build the installer. Although I did not change anything in the way I used the software, at some point it did not work any more. The error messages did not make any sense to me, so I gave up. I had tried to use pyexe, but that ran into an error immediately. So, I don't have a clue how I could produce a Windows installer. Sorry!
